github/event_types.go

```diff
@@ -165,6 +165,28 @@ type ProjectColumnChange struct {
 	} `json:"name,omitempty"`
 }
 
+// TeamChange represents the changes when a team has been edited.
+type TeamChange struct {
+	Description *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"description,omitempty"`
+	Name *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"name,omitempty"`
+	Privacy *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"privacy,omitempty"`
+	Repository *struct {
+		Permissions *struct {
+			From *struct {
+				Admin *string `json:"admin,omitempty"`
+				Push  *string `json:"push,omitempty"`
+				Pull  *string `json:"pull,omitempty"`
```

Comment:
Ah, I didn't notice that. I was going from most permissive to least, but it's better to follow the order of the documentation. I'll change that.